cypress/e2e/cloud/userProfile.test.ts

```diff
@@ -1,28 +1,3 @@
-// Utility Functions
```

Comment:
This change just moves this block lower in the file to make reviewing the variables it relies on more intuitive.